lib/codecs/adts.js

```diff
@@ -75,8 +75,11 @@ AdtsStream = function(handlePartialSegments) {
 
     // unpack any ADTS frames which have been fully received
     // for details on the ADTS header, see http://wiki.multimedia.cx/index.php?title=ADTS
-    while (i + 5 < buffer.length) {
 
+    // we use i + 3 here because we want to be able to parse the frame length
+    // from the header. If we don't have enough bytes to do that, than we
+    // definitely won't have a full frame.
+    while ((i + 3) < buffer.length) {
```

Comment:
Since we use i + 5 in determining frameLength below (and i + 6 below that), wouldn't we require at least that much?

Reply:
you're right, I only looked at the first byte that it asks for, I will update this.